default.nix

```diff
@@ -65,13 +84,13 @@ let
 
   samples = callPackages ./pkgs/samples { inherit debs cudaVersion autoAddOpenGLRunpathHook l4t cudaPackages; };
 
-  kernel = callPackage ./kernel { inherit (l4t) l4t-xusb-firmware; kernelPatches = []; };
+  kernel = callPackage ./kernel { inherit (l4t) l4t-xusb-firmware; kernelPatches = [ ]; };
```

Comment:
Could you remove these extraneous whitespace changes? They add a bunch of unneeded churn to the commits and make it difficult to review the PR.